back-end/chatbot-service/translation/translator.py

```
@@ -0,0 +1,16 @@
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)